README.md

```diff
@@ -1,15 +1,15 @@
 # Recursion Problems
 
 ## Definitions
-Define the following: 
+Define the following:
 
-- Recursion
-- Recursive Case
-- Base Case
-- Activation Chain/Stack
-- Activation Record/Call
-- Infinite Recursion/Stack Overflow/Stack too deep
-- Tail Recursion
+- Recursion - Recursion occurs when a method calls itself.
+- Recursive Case - A recursive case is the part of the method that contains the recursion.
+- Base Case - The base case is the part of the method that stops the recursion (and produces an answer I think).
+- Activation Chain/Stack - The activation chain are a cascade of steps the recursive method goes through. (They are the 'bubbles' I drew out to get the answers below.)
+- Activation Record/Call - The activation record is the memory area in the computer where local data for your current operation is stored.
+- Infinite Recursion/Stack Overflow/Stack too deep - This is a recursion that goes on infinitely.
+- Tail Recursion - (I know this was not covered, just read about it out of curiosity.) My guess from googling: it is a method that has its recursive component (i.e. the method) in the last line of the block.
```

Comment:
Close - it's where the last recursive call produces an answer and then that answer is served "up" all the recursive calls without any additional computations.